docs/api/session.md

```diff
@@ -698,6 +698,45 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
 })
 ```
 
+#### `ses.setDisplayMediaRequestHandler(handler)`
+
+* `handler` Function | null
+  * `request` Object
+    * `frame` [WebFrameMain](web-frame-main.md) - frame that is requesting access to media.
+    * `securityOrigin` String - origin of the page making the request.
+    * `videoRequested` Boolean - true if the web content requested a video stream.
+    * `audioRequested` Boolean - true if the web content requested an audio stream.
+    * `userGesture` Boolean - whether a user gesture was active when this request was triggered.
+  * `callback` Function
+    * `streams` Object
+      * `video` Object | [WebFrameMain](web-frame-main.md) (optional)
+        * `id` String - the id of the stream being granted.
+        * `name` String - the name of the stream being granted.
+      * `audio` Object | [WebFrameMain](web-frame-main.md) (optional)
+        * `id` String - the id of the stream being granted.
```

Comment:
So actually, I think this id can only be either `"loopback"`, `"loopbackWithMute"`, or a WebFrameMain, and the former two only work on Windows.

Further, when providing a WebFrameMain, there's also the option to ["disable local echo"](https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/web_contents_media_capture_id.h;l=43;drc=01af02b7cbaca23f29f88c441a3bed4790f1b3b9), the functionality of which I'm not totally sure about.

So... this should maybe be tweaked!

Reply:
I tweaked this.

- I now always disable local echo, since that seems to be Chromium's mostly-default.
- I updated the `audio` object to be `loopback`, `loopbackWithMute` or `WebFrameMain`.

Reply:
We've discovered that passing `false` allows us to retain audio output to speakers should we record the stream using `MediaRecorder`. I've created a feature ticket here:

https://github.com/electron/electron/issues/37293